cmd/rookflex/cmd/init.go

```diff
@@ -37,12 +39,18 @@ func init() {
 }
 
 func initPlugin(cmd *cobra.Command, args []string) error {
+	rookEnableSelinuxRelabeling, err := strconv.ParseBool(os.Getenv(agent.RookEnableSelinuxRelabelingEnv))
+	if err != nil {
+		//Don't throw an error, just default to true. This value was already validated by the operator.
```

Comment:
I think it would be good to log at least a warning when this happens.
Additionally please add a space after the `//`.